docs/router/framework/solid/quick-start.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Quick Start
+---
+
+If you're feeling impatient and prefer to skip all of our wonderful documentation, here is the bare minimum to get going with TanStack Router using both file-based route generation and code-based route configuration:
+
+## Using File-Based Route Generation
+
+File based route generation (through Vite, and other supported bundlers) is the recommended way to use TanStack Router as it provides the best experience, performance, and ergonomics for the least amount of effort.
+
+### Manual Setup
+
+Alternatively, you can manually setup the project using the following steps:
+
+#### Install TanStack Router, Vite Plugin
+
+```sh
+npm install @tanstack/solid-router
+npm install -D @tanstack/router-plugin
+# or
+pnpm add @tanstack/solid-router
+pnpm add -D @tanstack/router-plugin
+# or
+yarn add @tanstack/solid-router
+yarn add -D @tanstack/router-plugin
+# or
+bun add @tanstack/solid-router
+bun add -D @tanstack/router-plugin
+# or
+deno add npm:@tanstack/solid-router npm:@tanstack/router-plugin
+```
+
+#### Configure the Vite Plugin
+
+```tsx
+// vite.config.ts
+import { defineConfig } from 'vite'
+import solid from 'vite-plugin-solid'
```

Comment:
do we want to make a note about installing vite-plugin-solid?

Reply:
there's not note on the react vite plugin, so i think let's just keep the pages as similar as possible.

Reply:
good point i realized the other one didn't have it